web/src/engine/attachment/src/outputTargetForElement.ts

```diff
@@ -40,6 +40,10 @@ export function outputTargetForElement(Ltarg: HTMLElement) {
     }
   }
 
+  if (Ltarg._kmwAttachment == null) {
+    return null;
+  }
+
   // Step 2:  With the most likely host element determined, obtain the corresponding OutputTarget
   // instance.
   return Ltarg._kmwAttachment.interface;
```

Comment:
A null / undefined check here is pretty reasonable.  We can make it a bit more compact, though.

```suggestion
  // Step 2:  With the most likely host element determined, obtain the corresponding OutputTarget
  // instance.
  return Ltarg._kmwAttachment?.interface;
```